.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Mark or close stale issues and PRs'
+
+on:
+    schedule:
+        - cron: '00 * * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/stale@v3.0.14
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  days-before-issue-stale: 90
+                  days-before-pr-stale: 30
+                  days-before-close: 3
+                  stale-issue-message: 'This issue has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+                  stale-pr-message: 'This PR has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+                  close-issue-message: 'This issue was automatically closed due to being stale. Feel free to open a new one if you still experience this problem.'
```

Comment:
Is this what we want... to open a new issue? Or, should we suggest re-opening the existing issue?

Reply:
I like re-opening the existing issue better, to avoid duplication in search results.

Reply:
Sounds good I'll reword this.